postgraas_server/management_resources.py

```diff
@@ -137,6 +137,12 @@ def post(self):
         if not args['db_pwd']:
             abort(400, msg='The password may not be empty.')
 
+        if args['db_username'].startswith("postgres"):
+            abort(
+                406,
```

Comment:
I'm missing how this request (or the `db_username` parameter) relate to the **HTTP headers** `Accept`/`Accept-Charset`/...
At least that what I see status [406](https://tools.ietf.org/html/rfc7231#section-6.5.6) **should** be used for.

I'd suggest status [422](https://tools.ietf.org/html/rfc4918#section-11.2):
> [...] this error condition may occur if an XML request body contains well-formed (i.e., syntactically correct), but semantically erroneous, XML instructions.